src/app/views/common/monaco/Monaco.tsx

```diff
@@ -47,7 +48,7 @@ export function Monaco(props: IMonaco) {
             height={verbIsGet ? '80vh' : '350px'}
             // @ts-ignore
             value={body  ? body : ''}
-            language='json'
+            language={language ? language : 'json'}
```

Comment:
setting `json' as the default on line 34 is better, don't you think?

Reply:
Yes that would be better, but it interfered with Monaco updating it's state. Ie it would not rerender on receiving new props. That's why I am setting it explicitly.